app/assets/javascripts/pages/matomo_events.js

```diff
@@ -2,7 +2,7 @@
 document.addEventListener("DOMContentLoaded", function() {
   const clickOnLanding = document.getElementsByClassName("click-on-landing")
   if (clickOnLanding.length !== 0) {
-    clickOnLanding.forEach(function (item) {
+    Array.from(clickOnLanding).forEach(function (item) {
```

Comment:
Pour l'erreur Sentry `e.forEach is not a function` 
Testé sur staging ça a l'air de le faire